libs/designsystem/src/lib/components/calendar/calendar.component.scss

```diff
@@ -71,6 +63,10 @@ td {
   }
 }
 
+:not(:host(.has-year-navigator)) .month-navigator {
+  flex-grow: 1;
+}
+
```

Comment:
Consider readability by removing `:not(:host(.has-year-navigator))` and "resetting" the value inside the `:host(.has-year-navigator)` selector below